src/yamlfix/adapters.py

```diff
@@ -361,6 +362,25 @@ def fix(self, source_code: str) -> str:
 
         return source_code
 
+    @staticmethod
+    def _remove_extra_whitelines(match: Match[str]) -> str:
```

Comment:
I'd move this method below `_fix_whitelines` (so on line L670) so that it follows the top down normal reading flow. Reading it out of context feels weird and you need to jump down and then back up

Reply:
Done in [f61e191](https://github.com/lyz-code/yamlfix/pull/208/commits/f61e191ecee59d8b0a6fe3cc9a0934d58e12ff0d)